extensions/blocks/rating-star/index.js

```diff
@@ -27,7 +28,7 @@ export const settings = {
 		_x( 'rating', 'block search term', 'jetpack' ),
 		_x( 'review', 'block search term', 'jetpack' ),
 	],
-	category: supportsCollections() ? 'formatting' : 'jetpack',
+	category: supportsCollections() ? getCategoryWithFallbacks( 'text', 'formatting' ) : 'jetpack',
```

Comment:
@mkaz 🤔  What do you think ratings block should be? `text` category seems off…

```js
export const DEFAULT_CATEGORIES = [
	{ slug: 'text', title: __( 'Text' ) },
	{ slug: 'media', title: __( 'Media' ) },
	{ slug: 'design', title: __( 'Design' ) },
	{ slug: 'widgets', title: __( 'Widgets' ) },
	{ slug: 'embed', title: __( 'Embeds' ) },
	{ slug: 'reusable', title: __( 'Reusable blocks' ) },
];
```
[Default categories](https://github.com/WordPress/gutenberg/blob/89e34315314d7800c988301f06ae3d226f31bf44/packages/blocks/src/store/reducer.js#L34-L41)

Reply:
Maybe widgets?

Reply:
Widgets seems right-ish 🤷‍♂️  at least by ruling out all the other category options

Reply:
Done.